docs/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.py

```diff
@@ -0,0 +1,158 @@
+# is computed.  The 30 and 50mb U time series are also plotted.
+#
+
+##############################################################################
+# Datasets
+# --------
+#
+# GFS 0-24 hour forecasts for 10/2017 - 2/2018
+# ERA: 30 year climatology for EOFs and 10/2017 - 2/2018  
+#
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case runs the UserScript wrapper tool to run a user provided script,
+# in this case, stratosphere_qbo_driver.py to compute the zonal and meridional
+# mean, compute anomalies, create EOFs, create Phase diagrams, output U at 30 and
+# 50mb, and create time series of 30 and 50mb winds.  StatAnalysis is then run on 
+# the 30 and 50mb winds to compute the bias (ME).
+#
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# This use case does not loop but plots the entire time period of data.  The 
+# following tools are run once: UserScript, StatAnalysis
+# 
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/s2s_stratosphere/UserScript_fcstGFS_obsERA_StratosphereQBO.conf
+#
+
+#############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on user settings in the METplus configuration file. 
+# See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details. 
+#
+# **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
+#
+# If there is a setting in the MET configuration file that is currently not supported by METplus you'd like to control, please refer to:
+# :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
+#
+# **StatAnalysisConfig_wrapped**
+#
+# .. note:: See the :ref:`Stat-Analysis MET Configuration<series-analysis-met-conf>` section of the User's Guide for more information on the environment variables used in the file below:
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/met_config/STATAnalysisConfig_wrapped
+#
+
+##############################################################################
+# Python Embedding
+# ----------------
+#
+# This use case does not use python embedding.
+#
+
+##############################################################################
+# Running METplus
```

Comment:
I think this contains some of the old language (including the -c command) for this section. The streamlined version looks like the following:

> Pass the use case configuration file to the run_metplus.py script along with any user-specific system configuration files if desired:

> run_metplus.py /path/to/METplus/parm/use_cases/model_applications/marine_and_cryosphere/PointStat_fcstGFS_obsASCAT_satelliteWinds.conf /path/to/user_system.conf
> 
> See[ Running METplus](https://metplus.readthedocs.io/en/develop/Users_Guide/getting_started.html#running-metplus) for more information.